### PR TITLE
Improve DX, add local S3, mock user-service away so you don't need it

### PR DESCRIPTION
**Pull request by cxcorp**

### CODE REVIEW

File: .env-sample [hunk 1]
```diff
@@ -1,4 +1,4 @@
-PORT=9001
+PORT=9000
 PG_CONNECTION_STRING=postgresql://tarpisto:Su5hgVvoqUCRw5vrWmrW@localhost:9876/tarpisto
```
Comment (FinThunderstorm) [inline .env-sample]: Kept giving `Error: connect ECONNREFUSED ::1:5432
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1300:16)` if this was localhost, fixes by changing to `127.0.0.1`.

Comment (FinThunderstorm) [reply .env-sample]: Same thing with `AWS_S3_ENDPOINT`, changing to `127.0.0.1` fixes problems with `localhost`.

Comment (r-jheikkila) [reply .env-sample:2]: Are you running inside WSL2?

Comment (cxcorp) [reply .env-sample]: Are you using WSL2?

Comment (FinThunderstorm) [reply .env-sample]: M1 Mac

Comment (cxcorp) [reply .env-sample]: OK, changed localhost -> 127.0.0.1. Does the rest of it work alright?

Comment (FinThunderstorm) [reply .env-sample]: When doing initial testing, rest of it seems to work fine.

**Pull request merged**